lib/appsignal/span.rb

```diff
@@ -0,0 +1,84 @@
+module Appsignal
+  class Span
```

Comment:
```suggestion
  # @api private
  class Span
```

Mark as private so that users don't start using it yet if they stumble across it in our API docs.

Reply:
It's not really private, just experimental.